etc/layernames.pegjs

```diff
@@ -110,6 +112,9 @@ spec "Layer specification"
             name: layername.trim()
         };
     }
+    / _ {
```

Comment:
@pineapplespatula can you explain this section to me (since we're both trying to figure out htf pegjs works)